src/ECCX08.cpp

```diff
@@ -440,6 +440,113 @@ int ECCX08Class::lock()
   return 1;
 }
 
+
+int ECCX08Class::beginHMAC(uint16_t keySlot)
+{
+  // HMAC implementation is only for ATECC608
+  long ver = version() & 0x0F00000;
+  if (ver != 0x0600000) {
```

Comment:
local vars for describing what these values are doing

Reply:
Addressed in new commit